antlr-kotlin-tests/src/jsTest/kotlin/com/strumenta/antlrkotlin/test/grammars/TSqlGrammarJsTest.kt

```diff
@@ -0,0 +1,36 @@
+// This is needed as the IDE is not able to properly reference
+// the TSqlParser class as it is too big
+@file:Suppress("UNRESOLVED_REFERENCE")
+
+package com.strumenta.antlrkotlin.test.grammars
+
+import com.strumenta.antlrkotlin.test.GrammarTest
+import com.strumenta.antlrkotlin.test.generated.TSqlLexer
+import com.strumenta.antlrkotlin.test.generated.TSqlParser
+import org.antlr.v4.kotlinruntime.CharStream
+import org.antlr.v4.kotlinruntime.ParserRuleContext
+import org.antlr.v4.kotlinruntime.TokenStream
+
+// Note(Edoardo): move the test back to commonTest once
+//  the MethodTooLargeException has been solved on the JVM
+//  by the Kotlin team.
+//  FQN: org.jetbrains.org.objectweb.asm.MethodTooLargeException
+@Suppress("unused")
```

Comment:
Should we add a link to a YouTrack issue here?

Reply:
I still haven't created it, but I think we can add it later on anyway.

Reply:
Makes sense